### PR TITLE
K8 usecases 

### PR DESCRIPTION
**Pull request by sumitnagal**

### CODE REVIEW

File: charts/generic/k8-calico-node/experiment.yaml [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+      - apiGroups: ["","litmuschaos.io"]
+        resources: ["pods","configmaps","events","services","chaosengines","chaosexperiments","chaosresults","deployments","jobs"]
+        verbs: ["get","create","update","patch","delete","list"] 
+      - apiGroups: [""]
+        resources: ["nodes"]
+        verbs : ["get","list"]
+    labels:
+      name: k8-pod-delete    
+      app.kubernetes.io/part-of: litmus
+    image: "docker.artifactory.a.intuit.com/oicp/standard/external/chaostoolkit:latest"
```
Comment (navinjoy) [inline charts/generic/k8-calico-node/experiment.yaml]: the docker image should be updated to `litmuschaos/chaostoolkit:latest`

File: charts/generic/k8-kiam/experiment.yaml [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+      - apiGroups: ["","litmuschaos.io"]
+        resources: ["pods","configmaps","events","services","chaosengines","chaosexperiments","chaosresults","deployments","jobs"]
+        verbs: ["get","create","update","patch","delete","list"] 
+      - apiGroups: [""]
+        resources: ["nodes"]
+        verbs : ["get","list"]
+    labels:
+      name: k8-pod-delete    
+      app.kubernetes.io/part-of: litmus
+    image: "docker.artifactory.a.intuit.com/oicp/standard/external/chaostoolkit:latest"
```
Comment (navinjoy) [inline charts/generic/k8-kiam/experiment.yaml]: the docker image should be updated to litmuschaos/chaostoolkit:latest

File: charts/generic/k8-wavefront-collector/experiment.yaml [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+      - apiGroups: ["","litmuschaos.io"]
+        resources: ["pods","configmaps","events","services","chaosengines","chaosexperiments","chaosresults","deployments","jobs"]
+        verbs: ["get","create","update","patch","delete","list"] 
+      - apiGroups: [""]
+        resources: ["nodes"]
+        verbs : ["get","list"]
+    labels:
+      name: k8-pod-delete    
+      app.kubernetes.io/part-of: litmus
+    image: "docker.artifactory.a.intuit.com/oicp/standard/external/chaostoolkit:latest"
```
Comment (navinjoy) [inline charts/generic/k8-wavefront-collector/experiment.yaml]: the docker image should be updated to litmuschaos/chaostoolkit:latest

File: workflows/k8-kiam/workflow.yaml [hunk 4]
```diff
@@ -0,0 +1,148 @@
...
+                          - name: LABEL_NAME
+                            value: {{workflow.parameters.appLabel}}
+                          - name: APP_ENDPOINT
+                            value: {{workflow.parameters.appEndpoint}}
+                          - name: FILE
+                            value: {{workflow.parameters.fileName}}
+                          - name: REPORT
+                            value: 'true'
+                          - name: REPORT_ENDPOINT
+                            value: 'https://eventbus-e2e.intuit.com/v2/ip-quality-reliability-realtime-prf'
```
Comment (navinjoy) [inline workflows/k8-kiam/workflow.yaml]: This line should be removed and should be parameterized `{{workflow.parameters.reportEndpoint}}`

File: workflows/k8-service-kill/workflow.yaml [hunk 5]
```diff
@@ -0,0 +1,148 @@
...
+                          - name: LABEL_NAME
+                            value: {{workflow.parameters.appLabel}}
+                          - name: APP_ENDPOINT
+                            value: {{workflow.parameters.appEndpoint}}
+                          - name: FILE
+                            value: {{workflow.parameters.fileName}}
+                          - name: REPORT
+                            value: 'true'
+                          - name: REPORT_ENDPOINT
+                            value: 'https://eventbus-e2e.intuit.com/v2/ip-quality-reliability-realtime-prf'
```
Comment (navinjoy) [inline workflows/k8-service-kill/workflow.yaml]: This line should be removed and should be parameterized {{workflow.parameters.reportEndpoint}}

File: workflows/k8-wavefront-collector/workflow.yaml [hunk 6]
```diff
@@ -0,0 +1,148 @@
...
+                          - name: LABEL_NAME
+                            value: {{workflow.parameters.appLabel}}
+                          - name: APP_ENDPOINT
+                            value: {{workflow.parameters.appEndpoint}}
+                          - name: FILE
+                            value: {{workflow.parameters.fileName}}
+                          - name: REPORT
+                            value: 'true'
+                          - name: REPORT_ENDPOINT
+                            value: 'https://eventbus-e2e.intuit.com/v2/ip-quality-reliability-realtime-prf'
```
Comment (navinjoy) [inline workflows/k8-wavefront-collector/workflow.yaml]: This line should be removed and should be parameterized {{workflow.parameters.reportEndpoint}}

File: workflows/k8-calico-node/workflow.yaml [hunk 7]
```diff
@@ -0,0 +1,148 @@
...
+                          - name: LABEL_NAME
+                            value: {{workflow.parameters.appLabel}}
+                          - name: APP_ENDPOINT
+                            value: {{workflow.parameters.appEndpoint}}
+                          - name: FILE
+                            value: {{workflow.parameters.fileName}}
+                          - name: REPORT
+                            value: 'true'
+                          - name: REPORT_ENDPOINT
+                            value: 'https://eventbus-e2e.intuit.com/v2/ip-quality-reliability-realtime-prf'
```
Comment (navinjoy) [inline workflows/k8-calico-node/workflow.yaml]: This line should be removed and should be parameterized {{workflow.parameters.reportEndpoint}}

**Review by navinjoy — CHANGES_REQUESTED** [review]

File: charts/generic/k8-calico-node/experiment.yaml (hunk 1 above)
Comment (sumitnagal) [reply charts/generic/k8-calico-node/experiment.yaml]: Fixed

File: charts/generic/k8-kiam/experiment.yaml (hunk 2 above)
Comment (sumitnagal) [reply charts/generic/k8-kiam/experiment.yaml]: fixed

File: charts/generic/k8-wavefront-collector/experiment.yaml (hunk 3 above)
Comment (sumitnagal) [reply charts/generic/k8-wavefront-collector/experiment.yaml]: fixed

File: workflows/k8-kiam/workflow.yaml (hunk 4 above)
Comment (sumitnagal) [reply workflows/k8-kiam/workflow.yaml]: fixed

File: workflows/k8-service-kill/workflow.yaml (hunk 5 above)
Comment (sumitnagal) [reply workflows/k8-service-kill/workflow.yaml]: fixed

File: workflows/k8-wavefront-collector/workflow.yaml (hunk 6 above)
Comment (sumitnagal) [reply workflows/k8-wavefront-collector/workflow.yaml]: fixed

File: workflows/k8-calico-node/workflow.yaml (hunk 7 above)
Comment (sumitnagal) [reply workflows/k8-calico-node/workflow.yaml]: fixed

**Review by navinjoy — DISMISSED** [review]

**Pull request merged**